src/llmq/quorums.cpp

```diff
@@ -1100,13 +1103,75 @@ void CQuorumManager::StartCleanupOldQuorumDataThread(const CBlockIndex* pIndex)
         }
 
         if (!quorumThreadInterrupt) {
-            DataCleanupHelper(m_evoDb.GetRawDB(), dbKeysToSkip);
+            WITH_LOCK(cs_db, DataCleanupHelper(*db, dbKeysToSkip));
         }
 
         LogPrint(BCLog::LLMQ, "CQuorumManager::StartCleanupOldQuorumDataThread -- done. time=%d\n", t.count());
     });
 }
 
+void CQuorumManager::MigrateOldQuorumDB(CEvoDB& evoDb) const
+{
+    LOCK(cs_db);
+    if (!db->IsEmpty()) return;
+
+    const auto prefixes = {DB_QUORUM_QUORUM_VVEC, DB_QUORUM_SK_SHARE};
+
+    LogPrint(BCLog::LLMQ, "CQuorumManager::%d -- start\n", __func__);
+
+    CDBBatch batch(*db);
+    std::unique_ptr<CDBIterator> pcursor(evoDb.GetRawDB().NewIterator());
+
+    for (const auto& prefix : prefixes) {
+        auto start = std::make_tuple(prefix, uint256());
+        pcursor->Seek(start);
+
+        int count{0};
+        while (pcursor->Valid()) {
+            decltype(start) k;
+            CDataStream s(SER_DISK, CLIENT_VERSION);
+            CBLSSecretKey sk;
+
+            if (!pcursor->GetKey(k) || std::get<0>(k) != prefix) {
+                break;
+            }
+
+            if (prefix == DB_QUORUM_QUORUM_VVEC) {
+                if (!evoDb.GetRawDB().ReadDataStream(k, s)) {
+                    break;
+                }
+                batch.Write(k, s);
+            }
+            if (prefix == DB_QUORUM_SK_SHARE) {
+                if (!pcursor->GetValue(sk)) {
+                    break;
+                }
+                batch.Write(k, sk);
+            }
+
+            if (batch.SizeEstimate() >= (1 << 24)) {
+                db->WriteBatch(batch);
+                batch.Clear();
+            }
+
+            ++count;
+            pcursor->Next();
+        }
+
+        db->WriteBatch(batch);
+
+        LogPrint(BCLog::LLMQ, "CQuorumManager::%d -- %s moved %d\n", __func__, prefix, count);
```

Comment:
we have already the function name and file name in logs (if enabled), it seems exceeding.

Maybe use here some other human-friendly comment?

```suggestion
        LogPrint(BCLog::LLMQ, "Migrate old quorumdb -- %s moved %d\n", prefix, count);
```